src/NuGet.Services.EndToEnd/Support/Clients/V2V3SearchClient.cs

```diff
@@ -476,6 +476,7 @@ public class V3SearchPackage
             public List<V3VersionEntry> Versions { get; set; }
             public string LicenseUrl { get; set; }
             public string IconUrl { get; set; }
+            public string ReadmeUrl { get; set; }
```

Comment:
Could you please confirm whether we need this anywhere for readme tests?

Reply:
good catch, I forgot to delete.